### DISCUSSION TITLE
Moving discussions from repo to organization

### DISCUSSION THREAD

**Discussion by jonaski** [community/community#49239] [Discussions] [answered]
### Select Topic Area

Question

### Body

How do I move existing discussions from a repo to a organization?

**Comment by LindseyB** [comment, marked as answer]
If the repository is set as the target repository for the organization discussions it will automatically be moved. Alternatively, if it isn't you can move those discussions to the repository that is used as the target repository for organization discussions. 

https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization
